public/apps/configuration/app-router.tsx

```diff
@@ -267,20 +279,7 @@ export function AppRouter(props: AppDependencies) {
                   return <PermissionList {...props} />;
                 }}
               />
-              <Route
-                path={ROUTE_MAP.tenants.href}
-                render={() => {
-                  setGlobalBreadcrumbs(ResourceType.tenants);
-                  return <TenantList tabID={'Manage'} {...props} />;
-                }}
-              />
-              <Route
```

Comment:
Make sure `getTenancyRoutes` returns this Route as well.

Reply:
This is a hidden URL route (/tenantsConfigureTab) that brings you directly to the configuration tab of the tenants page. However, it is not connected to any UI component (i.e. if you click on the tenancy tab and click configure, the url doesn't change). Since it is a hidden URL that is not connected to any UI component I opted to remove it.

Reply:
Offline sync. @cwperks pointed out it is linked in the get-started page, which I did not know, so I added this back. I also updated the get-started page to not include the tenancy panel if not included in yaml